pkg/synopsysctl/cmd_get.go

```diff
@@ -38,6 +38,10 @@ var getOutputFormat string
 // Get Command flag for -selector functionality
 var getSelector string
 
+// Get Command flag for --all-namespaces functionality
+var getAllNamespaces bool
+var allNamespacesFlag = "--all-namespaces"
```

Comment:
Can you pls move it to Constants

Reply:
And also, apart from the get alerts, get blackducks, get opssights, all the remaining part in the function are same. Probably we can create a common function and call them.